src/bot/buttons.py

```diff
@@ -0,0 +1,34 @@
+from telegram import InlineKeyboardButton
+
+from constaints.callback_data import (VIEW_TASKS, CHANGE_CATEGORY,
```

Comment:
Немного не верный импорт и лучше импортировать сразу модуль а потом к нему обращаться `callback_data.VIEW_TASKS` 
```suggestion
from .constaints import callback_data
```